docs/.vuepress/plugins/dump-docs-data/docs-versions.js

```diff
@@ -66,6 +85,7 @@ async function readFromGitHub() {
   return {
     versions,
     latestVersion: versions[0],
+    patches,
```

Comment:
After removing the mocks, the generated patches are way beyond the limit of 6 MAJOR versions. Can you apply the limit for patches as well?
```json
{"patches":[["12.2",["12.2.0"]],["12.1",["12.1.3","12.1.2","12.1.1","12.1.0"]],["12.0",["12.0.1","12.0.0"]],["11.1",["11.1.0"]],["11.0",["11.0.1","11.0.0"]],["10.0",["10.0.0"]],["9.0",["9.0.2","9.0.1","9.0.0"]],["8.4",["8.4.0"]],["8.3",["8.3.2","8.3.1","8.3.0"]],["8.2",["8.2.0"]],["8.1",["8.1.0"]],["8.0",["8.0.0"]],["7.4",["7.4.2","7.4.0"]],["7.3",["7.3.0"]],["7.2",["7.2.2","7.2.1","7.2.0"]],["7.1",["7.1.1","7.1.0"]],["7.0",["7.0.3","7.0.2","7.0.1","7.0.0"]],["6.2",["6.2.2","6.2.1","6.2.0"]],["6.1",["6.1.1","6.1.0"]],["6.0",["6.0.1","6.0.0"]],["5.0",["5.0.2","5.0.1","5.0.0"]],["4.0",["4.0.0"]],["3.0",["3.0.0"]],["2.0",["2.0.0"]],["0.38",["0.38.1","0.38.0"]]]}
```

Reply:
Changed within https://github.com/handsontable/handsontable/pull/10079/commits/e539c2a3a296ae4735d921341bed747ab79092d7 by an extra `slice`.